README.md

```diff
@@ -1,82 +1,11 @@
-# OCI Artifacts
+# OCI Artifacts Merges with Image and Distribution Specs
```

Comment:
Recommend the title of the repository remain the same. Even with Artifacts becoming part of the Image and Distribution specification, is there a need to retain any of the past scope that is currently described within this README which is being replaced by the content below?

Depending on the decision, additional modifications to the content to follow